src/screenshots/index.js

```diff
@@ -1,56 +1,19 @@
 import { find } from 'lodash';
-import sanitizeFilename from 'sanitize-filename';
 import moment from 'moment';
 import Capturer from './capturer';
+import PathPattern from './path-pattern';
 
 export default class Screenshots {
-    constructor (path) {
-        this.enabled               = !!path;
-        this.screenshotsPath       = path;
-        this.testEntries           = [];
-        this.screenshotBaseDirName = Screenshots._getScreenshotBaseDirName();
-        this.userAgentNames        = [];
-    }
-
-    static _getScreenshotBaseDirName () {
-        var now = Date.now();
-
-        return moment(now).format('YYYY-MM-DD_hh-mm-ss');
```

Comment:
@miherlosev You also fixed another bug here: the hours were not in 24h format, but are now! 🎉🎉🎉 Thank you!